### ISSUE TITLE
Insufficient Testing Task Input Validation

### ISSUE THREAD

**Issue by jkva** [w3c/aria-at-app#392]
When clicking "Submit Results" at the end of a testing task, input is validated. In case of incorrect input, the following happens:

1) User focus is moved to the first "incorrect" input; and
2) "required" or "required: mark output" `span`s are placed within the input `label`s.

This is insufficient, given that:

* In the case of text input fields (for AT output transcription) they are not marked as `aria-required`;
* the "required" color contrast ratio is 3.7:1, below the required 4.5:1; and
* there is no overview of input errors available to the user.

![A screenshot showing "required" and "required: mark output" validation errors within input labels](https://user-images.githubusercontent.com/217264/145806024-7e405958-3ed1-4773-980d-988f2c0b651a.png)

Ideally: 

* user focus is moved to a section near the top of the test run;
* this section has a heading "Please address these input validation errors" and a list of errors;
* each item within this list is an in-page anchor which jumps to the offending input; and
* if color is used to indicate error, its color contrast is 4.5:1 or higher.

**Comment by jscholes** [comment]
@jkva I agree with this.  I would place focus on the first in-page link within that list of messages.
